.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish to NPM
```

Comment:
**Suggestion:** Should we add a prefix to this name, so that we know it's related to the publish flow?

Reason is that if you look into the repo we have a lot of workflows already, and they're sorted alphabetically. It would be a bit easier to understand what's related to the publish flow if it was all grouped together.

```suggestion
name: Publish: Publish to NPM
```

Note: do not forget to rename the file (to add the prefix too) in case we adopt this.

![image](https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/22061051/aa030697-d80e-47d3-b3da-2c8ebacc449d)

Reply:
I've changed the name of the job in the file for grouping, but I didn't change the file name since it already starts with publish and I think it would get weird to double it. Let me know if you think that we should still change the name for this one.